third_party/CMakeLists.txt

```diff
@@ -40,78 +40,77 @@ else()
 
   set(aws_cpp_sdk_SOURCE_DIR ${CMAKE_CURRENT_LIST_DIR}/aws-sdk-cpp)
   set(aws_cpp_sdk_INSTALL "${CMAKE_BINARY_DIR}/aws_cpp_sdk_INSTALL")
-  set(aws_cpp_sdk_INCLUDE_PATH "${aws_cpp_sdk_INSTALL}/include")
+  set(AWSSDK_INCLUDE_DIRS "${aws_cpp_sdk_INSTALL}/include")
 
   if(WIN32)
-    list(APPEND aws_cpp_sdk_LIBS "${aws_cpp_sdk_INSTALL}/lib/aws-cpp-sdk-transfer.lib")
-    list(APPEND aws_cpp_sdk_LIBS "${aws_cpp_sdk_INSTALL}/lib/aws-cpp-sdk-s3.lib")
-    list(APPEND aws_cpp_sdk_LIBS "${aws_cpp_sdk_INSTALL}/lib/aws-cpp-sdk-core.lib")
+    list(APPEND AWSSDK_LINK_LIBRARIES "${aws_cpp_sdk_INSTALL}/lib/aws-cpp-sdk-transfer.lib")
```

Comment:
One improvement can be done here:
You can do
```
set(AWSSDK_LIBS aws-cpp-sdk-transfer aws-cpp-sdk-s3 ...)
if (WIN32)
    set(suffix lib)
elseif(APPLE)
    set(suffix a)
else()
    set(suffix a)
endif()
foreach(lib ${AWSSDK_LIBS})
    list(APPEND AWSSDK_LINK_LIBRARIES "${aws_cpp_sdk_INSTALL}/lib/${lib}.${suffix}")
endforeach()
# Add special dependency

Reply:
Please use `AWSSDK_LIBRARIES` not `AWSSDK_LINK_LIBRARIES` if possible.

Reply:
updated. Please review.